packages/esm-ward-app/src/ward-workspace/admit-patient-form-workspace/admit-patient-form-workspace.workspace.tsx

```diff
@@ -0,0 +1,174 @@
+import React, { useCallback, useEffect, useMemo, useState } from 'react';
+import {
+  type DefaultWorkspaceProps,
+  ExtensionSlot,
+  usePatient,
+  type Patient,
+  useConfig,
+  showSnackbar,
+} from '@openmrs/esm-framework';
+import styles from './admit-patient-form.scss';
+import { useAdmissionLocation } from '../../hooks/useAdmissionLocation';
+import useWardLocation from '../../hooks/useWardLocation';
+import { useTranslation } from 'react-i18next';
+import { filterBeds } from '../../ward-view/ward-view.resource';
+import type { BedLayout } from '../../types';
+import { z } from 'zod';
+import { Controller, useForm } from 'react-hook-form';
+import { zodResolver } from '@hookform/resolvers/zod';
+import { Form, Row, Column, DropdownSkeleton, Dropdown, ButtonSet, Button } from '@carbon/react';
+import { InlineNotification } from '@carbon/react';
+import { assignPatientToBed, createEncounter } from '../../ward.resource';
+import type { WardConfigObject } from '../../config-schema';
+import { useInpatientRequest } from '../../hooks/useInpatientRequest';
+
+interface AdmitPatientFormWorkspaceProps extends DefaultWorkspaceProps {
+  patient: Patient;
+}
+
+const AdmitPatientFormWorkspace: React.FC<AdmitPatientFormWorkspaceProps> = ({
+  patient,
+  closeWorkspace,
+  closeWorkspaceWithSavedChanges,
+  promptBeforeClosing,
+}) => {
+  const { t } = useTranslation();
+  const { location } = useWardLocation();
+  const [isSubmitting, setIsSubmitting] = useState(false);
+  const { mutate: mutateInpatientRequest } = useInpatientRequest();
+  const { admissionEncounterTypeUuid } = useConfig<WardConfigObject>();
+  const [showErrorNotifications, setShowErrorNotifications] = useState(false);
+  const { isLoading, admissionLocation, mutate: mutateAdmissionLocation } = useAdmissionLocation();
```

Comment:
See my comment below, but unfortunately we need to account for what happens if the bed management is not installed.

You can see the feature flag we used... (and how we broke up the cases) in this component:

https://github.com/openmrs/openmrs-esm-patient-management/blob/main/packages/esm-ward-app/src/ward-view/ward-view.component.tsx

Reply:
Hi @mogoodrich!
Yes, the check is present in the root component of the ward app and in case the module isn't present, the ward app will not render and this whole flow will not render too.

Hence I believe the assumption is right that the bed management will be available if the user is seeing the form.

Reply:
As an FYI to others, Vineet and I discussed this on standup and clarified how this form can be available even if the bed management module is not installed.